frontend/packages/console-app/src/components/terminal/Terminal.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { RootState } from '@console/internal/redux';
+import * as UIActions from '@console/internal/actions/ui';
+import CloudShellDrawer from '../cloud-shell/CloudShellDrawer';
+import CloudShellBody from '../cloud-shell/CloudShellBody';
+
+type TerminalContentProps = {
+  isTerminalExpanded: boolean;
+  toggleTerminal: any;
+};
+
+const TerminalContent: React.FC<TerminalContentProps> = ({
+  isTerminalExpanded,
+  toggleTerminal,
+}) => (
+  <CloudShellDrawer open={isTerminalExpanded} onClose={() => toggleTerminal()}>
+    <CloudShellBody />
+  </CloudShellDrawer>
+);
+
+const terminalStateToProps = ({ UI }: RootState) => ({
+  isTerminalExpanded: !!UI.getIn(['terminal', 'isExpanded']),
```

Comment:
Why double `!!`? `isExpanded` should be a boolean value or have `open, close` state.

Reply:
updated